proxy/http/HttpSM.h

```diff
@@ -196,6 +196,13 @@ class PendingAction
   {
     return pending_action;
   }
+  void
+  try_clear(Action *current_action)
```

Comment:
`clear_if`? `clear_if_action_is`?

Reply:
+1 on `clear_if_action_is`. `try_clear` reminds me of `try_lock` and make me want to put it into if statement.

Reply:
Ok will change to clear_if_action_is.